main.py

```diff
@@ -0,0 +1,248 @@
+import random
+
+suits = ('Hearts', 'Diamonds', 'Spades', 'Clubs')
+ranks = ('Two', 'Three', 'Four', 'Five', 'Six', 'Seven', 'Eight', 'Nine', 'Ten', 'Jack', 'Queen', 'King', 'Ace')
+values = {'Two':2, 'Three':3, 'Four':4, 'Five':5, 'Six':6, 'Seven':7, 'Eight':8, 
+            'Nine':9, 'Ten':10, 'Jack':11, 'Queen':12, 'King':13, 'Ace':14}
+playing = True
+
+class Card():
+    
+    def __init__(self,suit,rank):
+        self.suit = suit
+        self.rank = rank
+            
+    def __str__(self):
+        return self.rank + " of " + self.suit
+    
+    
+class Deck():
+    
+    def __init__(self):
+        self.deck = [] 
+        for suit in suits:
+            for rank in ranks:
+                self.deck.append(Card(suit,rank))
+                
+       
+    def __str__(self):
+        deck_comp = '' 
+        for card in self.deck:
+            deck_comp += '\n' + card.__str__()
+        return "The deck has: " + deck_comp
+
+    def shuffle(self):
+        random.shuffle(self.deck)
+        
+    def deal(self):
+        single_card =  self.deck.pop()
+        return single_card
+
+
+
+class Hand():
+    def __init__(self):
+        self.cards = []  # start with an empty list as we did in the Deck class
+        self.value = 0   # start with zero value
+        self.aces = 0    # add an attribute to keep track of aces
+    
+    def add_card(self,card):
```

Comment:
The add_card method has multiple functionalities. The method should have a single responsibility.